Triangle/Tasks.cs

```diff
@@ -16,6 +16,11 @@ public static class Tasks
                 .ToDictionary(group => group.Key, group => group.ToList());
         }
 
+        /// <summary>
+        /// This function reads objects of ColoredTriangle class from file and sorts them into SortedList by perimeter
```

Comment:
This line is too long, replace it with "This function reads ColoredTriangle objects and puts them into a sorted list. <new_line> The key of the sorted list is the perimeter of the corresponding triangle"